pkg/operatorstatus/operatorstatus.go

```diff
@@ -58,18 +59,19 @@ type ClusterOperatorWrapper interface {
 }
 
 func (o *ClusterOperatorHandler) UpdateOperatorStatus(cfg *v1.Config) error {
-
 	var err error
-	failing, stateForPending, msgForFailing := cfg.ClusterOperatorStatusFailingCondition()
+	failing, msgForProgressing, msgForFailing := cfg.ClusterOperatorStatusFailingCondition()
 	err = o.setOperatorStatus(configv1.OperatorFailing,
 		failing,
 		msgForFailing,
-		v1.GitVersionString())
+		"")
```

Comment:
I know this has been a moving target, but at one point, the direction was to include the version we failed to get to ... I'll see if the doc we reference before has changed, but was this a conscious choice?

Reply:
That should still happen.  This argument is only used to set the clusteroperator.status.version field, it's not used to generate the failing message, which is where you put the "failed to achieve version X" message, right?

The change here is we only set clusteroperator.status.version to the operator's current version when we are setting available=true.

Until available=true, we are considered to be at the previous version.

Reply:
My testing of the PR is not showing what you say:

```
status:
  conditions:
  - lastTransitionTime: 2019-02-28T18:00:17Z
    message: 'Samples installation successful at '
    status: "True"
    type: Available
  - lastTransitionTime: 2019-02-28T18:01:10Z
    message: 'Samples installation successful at '
    status: "False"
    type: Progressing
  - lastTransitionTime: 2019-02-28T18:00:14Z
    status: "False"
    type: Failing
  extension: null
  relatedObjects: null
  versions: null
```

My recollection was in fact correct.  You need to pass the version in that parameter.

Reply:
I still don't think so.  That argument is not used to set the message.  It wasn't before and it's not now.

Reply:
Nope I was wrong ... needed to update my deployment